.appveyor.yml

```diff
@@ -33,6 +33,14 @@ environment:
       BUILD_STATIC : ON
       PYTHON_VERSION: "3.6"
       PYTHON_ARCH: "64"
+      ENABLE_PYTHON : OFF
+      PACK: ON
+
+    - BLAS_VERSION: Intel10_64lp_seq
+      BUILD_STATIC : ON
+      PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "64"
+      ENABLE_PYTHON : ON
       PACK: ON
```

Comment:
PACK=ON means upload package? if so we have now 2 pack=ON?

Reply:
yep. my fault